src/modules/0.0.11/aws_cloudwatch/index.ts

```diff
@@ -21,10 +16,10 @@ const getLogGroups = paginateBuilder<CloudWatchLogs>(
 const getLogGroup = async (client: CloudWatchLogs, groupName: string) => (
   await getLogGroups(client)
 ).find(lg => lg.logGroupName === groupName);
-const deleteLogGroup = crudBuilder<CloudWatchLogs>(
+const deleteLogGroup = crudBuilderFormat<CloudWatchLogs, 'deleteLogGroup', undefined>(
   'deleteLogGroup',
-  (logGroupName: string) => ({ logGroupName, }),
-  (_lg: DeleteLogGroupCommandOutput) => undefined,
+  (logGroupName) => ({ logGroupName, }),
+  (_lg) => undefined,
```

Comment:
nit: This one (and all the ones returning  `undefined`) could just be `crudBuilder2` instead of `crudBuilderFormat`, right?

Reply:
So I used the format path here (which was baked into `crudBuilder`) to mimic the exact behavior of the AWS Gateway function it is replacing.

I don't mind making a change like that, but I have been trying to keep behavior-modifying changes to a minimum in this PR (only making them where the improved type safety tells me there's a bug in the code we missed before).